src/nfa/operations.cc

```diff
@@ -749,6 +778,7 @@ Simlib::Util::BinaryRelation Mata::Nfa::Algorithms::compute_relation(const Nfa&
     }
 }
 
+//All these parameters are making this a monster. state_map optional, params too (or remove), kick out trim_input and don't trim.
```

Comment:
You have to look at function declaration for specification of optional parameters. All of these are optional except for an automaton `aut`.

Reply:
okok. Bu we will kick out trim_input, right?

Reply:
Yes.